### ISSUE TITLE
[C3M0_B] Stierlitz does not spawn? Bomb not reachable?

### ISSUE THREAD

**Issue by r3538987** [Realm667/WolfenDoom#1327]
When you find "treasure room", he mentions that guards have been warned and you need to return. I'm expecting that it means to return to him, also map marker shows toward his initial position. 
Once you manage to get there, Stierlitz is not there anymore, and you also cannot grab bomb. 
![image](https://github.com/Realm667/WolfenDoom/assets/33725345/c4cae0dd-b5ce-41ee-b0b5-a6d048c6024e)
I'm thinking maybe I didn't completed certain task, but once alarm goes on, then reach him without seeing by guards for me was impossible. 
When this didn't worked out, I checked YT videos, and ran toward exit to trigger last cutscene.

**Comment by Talon1024** [comment]
I think that's a problem with the Russian translation. Here's the original English text:

Stierlitz  
Captain, they blew our cover! Get out of here now! Wehrmacht soldiers are pounding on my door, we need to... \*zzzZzzzZZZZ\*

**Comment by Username-N00b-is-not-available** [comment]
The Russian string in question is
>Штирлиц
>Капитан, мы раскрыты! Уходите — немедленно! Ко мне в дверь ломятся солдаты, нам нужно... *з-З-з-З-З-З*

It is almost a literal translation of the English message, he says `уходите`.

The bomb is decorative: Stierlitz had a mission to destroy the complex but decided against it, because there were signs of the Amber Room presence there.

**Comment by r3538987** [comment]
Yep, probably that was just my misunderstanding. I had this understood as, > check if he needs help > and then leave...
So we can surely close this.

**Comment by Talon1024** [comment]
This gives me an idea, however. Stierlitz could use his bomb to blow a hole in the wall and escape that way.

**Comment by r3538987** [comment]
For me it also wasn't clear where you had to escape as well. Had to watch videos :octocat:

**Comment by x3003** [comment]
This issue has been resolved in https://github.com/Realm667/WolfenDoom/issues/1554
You can find Stierlitz later in the camp, so he didn't escape. And the bomb has been seized, so it's not there anymore when Ryan got his last message.
